src/signals/triage_signals.R

```diff
@@ -140,21 +146,38 @@ preview_campaign_urls <- function(campaign_urls) {
 #' @param df Signals data frame
 #' @param fn_signals File name to the signals data
 #' @param test Whether or not the signals were in the test folder.
-approve_signals <- function(df, fn_signals, test) {
+approve_signals <- function(df, fn_signals, test, user_command = "init") {
```

Comment:
Actually maybe this should not be an argument of the function, but just a variable that's initiated in the first line of the function. Whatever value the variable it's initiated with it will be overwritten by the user keyboard input so I am thinking that's there is not really a point in having it as function argument

Reply:
removing as arg makes sense i think